### PR TITLE
Update duckdns.markdown

### PR DESCRIPTION
**Pull request by davaymne**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @davaymne, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18004).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by davaymne** [comment]
Correction in duckdns doc

File: source/_integrations/duckdns.markdown [hunk 1]
```diff
@@ -29,7 +29,7 @@ duckdns:
 
 {% configuration duckdns %}
   domain:
-    description: Your duckdns subdomain (without the `.duckdns.org` suffix).
```
Comment (frenck) [inline source/_integrations/duckdns.markdown:32]: I don't think this change is correct. The DuckDNS API expects a sudomain, not a full domain.

Unfortunately, this PR does not explain the reasoning behind this change in its description. Could you elaborate?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
No response from the author on the question in a month. Closing the PR for now.

Nevertheless, thank you for willing to contribute :+1:

**Pull request closed without merging**